### PR TITLE
Docs/new/baremetal quickstart

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Whole new section and various fixes.

### CODE REVIEW

File: product_docs/docs/pgd/5/quickstart/quick_start_bare.mdx [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+
+To leave the SQL client, enter `exit`.
+
+### Using PGD CLI
+
+The pgd utility, also known as the PGD CLI, lets you control and manage your Postgres Distributed cluster. It's already installed on the node. 
+
+You can use it to check the cluster's health by running `pgd check-health`:
+
+```console
```
Comment (josh-heyer) [inline product_docs/docs/pgd/5/quickstart/quick_start_bare.mdx:314]: consider using 
```
command
__OUTPUT__
output
``` 
format for these as well

**Review by josh-heyer — APPROVED** [review]
This went surprisingly well - I followed the walkthrough on Azure VMs without a hitch. The ssh-copy-id/ssh-keyscan dance is... Tedious, but having created the table of IPs to copy from at the start helped. I do wish there is some way we could use the information already in config.yaml to eliminate these steps.

File: product_docs/docs/pgd/5/quickstart/quick_start_bare.mdx (hunk 1 above)
Comment (josh-heyer) [reply product_docs/docs/pgd/5/quickstart/quick_start_bare.mdx:314]: eh, no, that obscures what's being done too much. Leave it as-is for now.

**Review by josh-heyer — DISMISSED** [review]
Made a few tweaks & corrections to the connection URL stuff; otherwise, LGTM!

File: product_docs/docs/pgd/5/quickstart/index.mdx [hunk 2]
```diff
@@ -1,32 +1,34 @@
...
+## Quick start
 
 Postgres Distributed (PGD) is a multi-master replicating implementation of Postgres designed for high performance and availability. You can create database clusters made up of many bidirectionally synchronizing database nodes. The clusters can have a number of proxy servers that direct your query traffic to the most available nodes, adding further resilience to your cluster configuration.
  
 PGD is very configurable. To quickly evaluate and deploy PGD, use this quick start. It'll get you up and running with a fully configured PGD cluster using the same tools that you'll use to deploy to production. This quick start includes:
 
 * A short introduction to Trusted Postgres Architect (TPA) and how it helps you configure, deploy, and manage Postgres Distributed
-* A guide to selecting Docker or AWS quick starts
-  * The Docker quick start 
+* A guide to selecting Docker, Linux hosts or AWS quick starts
```
Comment (drothery-edb) [inline product_docs/docs/pgd/5/quickstart/index.mdx:24]:
```suggestion
* A guide to selecting Docker, Linux hosts, or AWS quick starts
```

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**